src/shared/components/modlog.tsx

```diff
@@ -83,10 +84,11 @@ interface ModlogState {
   res: Option<GetModlogResponse>;
   communityId: Option<number>;
   communityMods: Option<CommunityModeratorView[]>;
+  communityName: Option<string>,
   page: number;
   siteRes: GetSiteResponse;
   loading: boolean;
-  filter_action?: ModlogActionType,
+  filter_action: ModlogActionType,
   filter_user?: number,
```

Comment:
I highly recommend using `filter_user_id: Option<number>`, and changing all the other typescript optionals to rust-style Option's. The only case where I've found it doesn't work perfectly, is when we have to use derivedState.